public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -451,6 +470,33 @@ export class PanelQueryRunner {
   getLastRequest(): DataQueryRequest | undefined {
     return this.lastRequest;
   }
+
+  shouldAddErrorWhenDatasourceVariableIsMultiple(
+    datasource: DataSourceRef | DataSourceApi | null,
+    scopedVars: ScopedVars | undefined
+  ): boolean {
+    let addWarningMessageMultipleDatasourceVariable = false;
+
+    //If datasource is a variable
+    if (datasource?.uid?.startsWith('${')) {
+      const variableName = this.templateSrv.getVariableName(datasource.uid) ?? '';
```

Comment:
you do not need this call, you can use variable.name instead (variable is passed as second argument in the function you pass to replace)